### ISSUE TITLE
`ValueError: max() arg is an empty sequence` on gen_graph.py

### ISSUE THREAD

**Issue by SlaterLatiao** [Dongdongshe/K-Scheduler#13]
I followed the instructions on [https://github.com/Dongdongshe/K-Scheduler/blob/main/afl_integration/build_example/README.md](https://github.com/Dongdongshe/K-Scheduler/blob/main/afl_integration/build_example/README.md) to build harfbuzz with K-scheduler,  and running the gen_graph.py script resulted in an error:
```
Traceback (most recent call last):
  File "/home/zenong/fuzz/K-Scheduler/afl_integration/build_example/gen_graph.py", line 250, in <module>
    max_score = max(k.scores())
ValueError: max() arg is an empty sequence
```
I checked the [IR file](https://github.com/Dongdongshe/K-Scheduler/files/10799991/target_fix.ll.zip) and made sure it was compiled with LLVM coverage sanitizer ( `__sanitizer_cov_trace_pc_guard` exist in the IR).
I used Python3.10 with the following networkit version.
```
zenong@3ac4dd221bf1:~/fuzz$ python3 -m pip install networkit
Requirement already satisfied: networkit in /home/zenong/.local/lib/python3.10/site-packages (10.0)
Requirement already satisfied: numpy in /home/zenong/.local/lib/python3.10/site-packages (from networkit) (1.24.2)
Requirement already satisfied: scipy in /home/zenong/.local/lib/python3.10/site-packages (from networkit) (1.10.1)
```

**Comment by Dongdongshe** [comment]
1. Double-check if your local repo contains the latest commit. 
2. If you were building a non-wrapper version, please follow the tutorial at https://github.com/Dongdongshe/K-Scheduler/blob/main/afl_integration/build_example/README(non_wrapper%20based%20program).md
3. If you still have problems, please email me or directly reply here with your error replication package (program source code, llvm version). I am happy to help.

**Comment by SlaterLatiao** [comment]
Thanks for your prompt response! I apologize for the delay as I missed the reminder for this issue. Upon investigation, I discovered that the cause of the problem was a typo in the folder name `cfg_out_harfbuzz` when I executed the command `cd .. && python ./gen_graph.py ./harfbuzz_afl_asan_fix.ll cfg_out_harfbuzz`. As a result, the script was unable to find the folder and generated an empty graph.
